packages/atomic/create-atomic/src/default/default-search-interface.ts

```diff
@@ -0,0 +1,39 @@
+export default `<atomic-search-interface>
```

Comment:
Pretty sure you could get nicer linting etc if you just change to .tsx

Reply:
It's a string literal, not JSX. It doesn't give me better lint/format

Reply:
I could change these files to html and import it as a string perhaps

Reply:
Worth a try :D

Reply:
Actually don't think I can do this with only the Typescript compiler, going to need esbuild or rollup, webpack, etc.

Reply:
Yeah, so let's keep it as is for now 😅